samples/automl/automlVisionDataset.js

```diff
@@ -0,0 +1,399 @@
+    },
+  };
+
+  // Import the dataset from the input URI.
+  client
+    .importData({name: datasetFullId, inputConfig: inputConfig})
+    .then(responses => {
+      const operation = responses[0];
+      console.log(`Processing import...`);
+      return operation.promise();
+    })
+    .then(responses => {
+      // The final result of the operation.
+      if (responses[2].done === true) console.log(`Data imported.`);
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_vision_importData]
+}
+
+function exportData(projectId, computeRegion, datasetId, outputUri) {
+  // [START automl_vision_exportData]
+  const automl = require(`@google-cloud/automl`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const datasetId = `Id of the dataset`;
+  // const outputUri = `Google Cloud Storage URI for the export directory, e.g. “gs://myproject/output”;`
+
+  // Get the full path of the dataset.
+  const datasetFullId = client.datasetPath(projectId, computeRegion, datasetId);
+
+  // Set the output URI
+  const outputConfig = {
+    gcsDestination: {
+      outputUriPrefix: outputUri,
+    },
+  };
+
+  // Export the data to the output URI.
+  client
+    .exportData({name: datasetFullId, outputConfig: outputConfig})
+    .then(responses => {
+      const operation = responses[0];
+      console.log(`Processing export...`);
+      return operation.promise();
+    })
+    .then(responses => {
+      // The final result of the operation.
+      if (responses[2].done === true) console.log(`Data exported.`);
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_vision_exportData]
+}
+
+function deleteDataset(projectId, computeRegion, datasetId) {
+  // [START automl_vision_deleteDataset]
+  const automl = require(`@google-cloud/automl`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const datasetId = `Id of the dataset`;
+
+  // Get the full path of the dataset.
+  const datasetFullId = client.datasetPath(projectId, computeRegion, datasetId);
+
+  // Delete a dataset.
+  client
+    .deleteDataset({name: datasetFullId})
+    .then(responses => {
+      const operation = responses[0];
+      return operation.promise();
+    })
+    .then(responses => {
+      // The final result of the operation.
+      if (responses[2].done === true) console.log(`Dataset deleted.`);
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_vision_deleteDataset]
+}
+
+require(`yargs`)

```

Comment:
@fhinkel @jmdobry are we still doing the `yargs` things for our samples?  I thought we were moving away from that?